src/Symfony/Component/Yaml/Parser.php

```diff
@@ -51,6 +52,31 @@ public function __construct()
     }
 
     /**
+     * Parses a YAML file into a PHP value.
+     *
+     * @param string $filename A string containing YAML
+     * @param int    $flags    A bit field of PARSE_* constants to customize the YAML parser behavior
+     *
+     * @return mixed The YAML converted to a PHP value
+     *
+     * @throws ParseException If the file could not be read or the YAML is not valid
+     */
+    public function parseFile($filename, $flags = 0)
+    {
+        if (!is_file($filename)) {
+            throw new ParseException(sprintf('File "%s" does not exist.', $filename));
+        }
+
+        if (!is_readable($filename)) {
+            throw new ParseException(sprintf('File "%s" cannot be read.', $filename));
+        }
+
+        $this->filename = $filename;
+
+        return $this->parse(file_get_contents($filename, $flags));
```

Comment:
The parenthesis are at the wrong place, `$flags` is an argument of the parse `method` not the `file_get_contents` function.